lib/livebook_web/live/settings_live/env_var_component.ex

```diff
@@ -0,0 +1,101 @@
+defmodule LivebookWeb.SettingsLive.EnvVarComponent do
+  use LivebookWeb, :live_component
+
+  alias Livebook.Settings
+  alias Livebook.Settings.EnvVar
+
+  @impl true
+  def mount(socket) do
+    env_var = %EnvVar{}
+    operation = :new
+    changeset = Settings.change_env_var(env_var)
+
+    {:ok, assign(socket, changeset: changeset, env_var: env_var, operation: operation)}
+  end
+
+  @impl true
+  def update(assigns, socket) do
+    {env_var, operation} =
+      unless assigns.env_var,
+        do: {%EnvVar{}, :new},
+        else: {assigns.env_var, :edit}
```

Comment:
unless with else is always confusing :D

```suggestion
      if assigns.env_var,
        do: {assigns.env_var, :edit},
        else: {%EnvVar{}, :new}
```

Reply:
I can't disagree 🐱